### DISCUSSION TITLE
Repititive Use of a Rule

### DISCUSSION THREAD

**Discussion by w0o0o0oah** [taocpp/PEGTL#75] [Q&A] [answered]
If I have a rule contained in a plus statement how do I get that rule to apply method to occur on every successive input it succeeds on rather than just once for the entirety of the input that matches with the rule?

**Comment by d-frey** [comment, marked as answer]
You mean something like:

```cpp
struct A : seq< whatever > {};
struct B : plus< A > {};
```

?

If you specialize an action for `B`, it will receive the whole input that matches in one go. If you want to have your action called for each `A` you'd simply specialize an action for `A`.

**Comment by w0o0o0oah** [comment]
Yes that works. I was trying to combine both of those lines.
